main.py

```diff
@@ -1,10 +1,49 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as f:
+        players_data = json.load(f)
+
+    for nickname, player_data in players_data.items():
+        race_name = player_data["race"]["name"]
```

Comment:
you don't really need to create extra variables for it since you use it only once